Makefile

```diff
@@ -63,8 +63,8 @@ install:
 
 .PHONY: changelog
 changelog:
-	@$(CURRENT_DIR)/scripts/changelog.sh
+	@$(shell pwd)/scripts/changelog.sh
```

Comment:
@mccurdyc FYI. Assuming `CURRENT_DIR` may be specific to you environment or Arch linux? Just checking you're ok with using `pwd` here instead?